packages/evergreen-badges/src/styles/badge-appearances.js

```diff
@@ -1,12 +1,21 @@
+import colors from 'evergreen-colors'
+
+const baseStyle = {
+  boxSizing: 'border-box',
+  textDecoration: 'none',
+  textTransform: 'uppercase',
+  borderRadius: 2,
+}
+
 const BadgeAppearances = {
-  neutral: {},
-  red: {},
-  green: {},
-  blue: {},
-  purple: {},
-  pink: {},
-  yellow: {},
-  turquoise: {},
+  neutral: { ...baseStyle },
```

Comment:
I am actually thinking it makes most sense to do something like. This also has the benefit of making it easier to use in a Sketch generation tool.

```js
import colors from 'evergreen-colors'

const badgeColors = Object.keys(colors).filter(c => c !== 'white')

const getSolidBadgeStyle = (colorName) => {
  return {
    backgroundColor: badgeColors[colorName]['500'],
    color: 'white',
  }
}

const getDefaultBadgeStyle = (colorName) => {
  return {
    backgroundColor: badgeColors[colorName]['15A'],
    color: badgeColors[colorName]['1000'],
  }
}

const BadgeAppearances = {
  default: badgeColors.map(getDefaultBadgeStyle),
  solid: badgeColors.map(getSolidBadgeStyle),
}

export default BadgeAppearances
```

Reply:
Made some mistakes, should be:
```
const getSolidBadgeStyle = (colorName) => {
  return {
    backgroundColor: colors[colorName]['500'], // colors instead of badgeColors
    color: 'white',
  }
}

const getDefaultBadgeStyle = (colorName) => {
  return {
    backgroundColor: colors[colorName]['15A'], // colors instead of badgeColors
    color: colors[colorName]['1000'], // colors instead of badgeColors
  }
}

const defaultAppearances = {}
const solidAppearances = {}

badgeColors.forEach(colorName => {
  defaultAppearances[colorName] = getDefaultBadgeStyle(colorName)
  solidAppearances[colorName] = getSolidBadgeStyle(colorName)
})

const BadgeAppearances = {
  default: defaultAppearances,
  solid: solidAppearances,
}
```

Reply:
yeah figured I'd text you about this.  I was like, did he mean to use an array?  lol.  see updated PR.